### PR TITLE
feat: run kilt locally

### PR DESCRIPTION
**Pull request by francesco-racciatti**
Signed-off-by: Francesco Racciatti <francesco.racciatti@sysdig.com>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

/kind feature

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area build

> /area tests

> /area proposals

**What this PR does / why we need it**:
This PR makes it possible to run Kilt locally.

### CODE REVIEW

File: runtimes/cloudformation/cmd/handler/main.go [hunk 1]
```diff
@@ -77,12 +99,32 @@ func ConfigClosure() interface{} {
...
+		}
+
+		err = os.WriteFile(os.Getenv("KILT_OUT_TEMPLATE"), result, 0644)
+		if err != nil {
+			panic("cannot write dst file " + os.Getenv("KILT_OUT_TEMPLATE"))
+		}
+
+	default:
+		lambda.Start(ConfigClosure())
+	}
```
Comment (gnosek) [inline runtimes/cloudformation/cmd/handler/main.go:129]: Hmm how about we move the GetConfig call before the switch, and then:
* pass the variable into PatchLocalFile
* inline ConfigClosure into the default case (it becomes a single statement)

**Review by gnosek — COMMENTED** [review]
Minor nit, feel free to ignore it, LGTM otherwise

File: runtimes/cloudformation/cmd/handler/main.go (hunk 1 above)
Comment (francesco-racciatti) [reply runtimes/cloudformation/cmd/handler/main.go:129]: Uh sure

**Review by gnosek — APPROVED** [review]

**Pull request merged**